torch/csrc/autograd/python_variable.cpp

```diff
@@ -165,7 +167,7 @@ static int THPVariable_clear(THPVariable *self)
     // objects stay live, buster!  See
     // https://github.com/pytorch/pytorch/issues/22884 for an example of
     // this actually showing up.
-    set_pyobj(self->cdata, nullptr);
+    tensor.unsafeGetTensorImpl()->unchecked_clear_pyobj(self_interpreter.get());
```

Comment:
i see.  so, this seems ok.  It preserves the invariant that once on an interp, always on that interp.

Reply:
Yup. Clears pyobj but not interpreter tag